docs/integrations/gerrit/locations.md

```diff
@@ -0,0 +1,36 @@
+---
+id: locations
+title: Gerrit Locations
+sidebar_label: Locations
+description: Integrating source code stored in Gerrit into the Backstage catalog
+---
+
+The Gerrit integration supports loading catalog entities from Gerrit hosted gits. Entities can
+be added to [static catalog configuration](../../features/software-catalog/configuration.md),
+or registered with the
+[catalog-import](https://github.com/backstage/backstage/tree/master/plugins/catalog-import)
+plugin.
+
+## Configuration
+
+To use this integration, add configuration to your root `app-config.yaml`:
+
+```yaml
+integrations:
+  gerrit:
+    - host: gerrit.company.com
+      apiBaseUrl: gerrit.company.com/gerrit
+      username: ${GERRIT_USERNAME}
+      password: ${GERRIT_PASSWORD}
+```
+
+Directly under the `gerrit` key is a list of provider configurations, where
+you can list the Gerrit instances you want to fetch data from. Each entry is
+a structure with up to four elements:
+
+- `host`: The host of the Gerrit instance, e.g. `gerrit.company.com`.
+- `apiBaseUrl`: The base url of the Gerrit API. This would typically be the address
```

Comment:
Does it make sense to have a default implemented for it? Or can this be too varying between different installations?

At the config schema (config.d.ts) it is marked as optional but not here. And at config.ts/ the integration config class, it is require (which is good) and read as required from the config.

At least one places seems to be wrong.

Reply:
Nice catch! I've set it to optional everywhere now. It will default to just using the address specified in "host" and plus a https prefix.